assets/auto-vj-uri.html

```diff
@@ -108,6 +150,19 @@
                 if(meta.tempo) {
                     player.playbackRate = meta.tempo;
                     player.defaultPlaybackRate = meta.tempo;
+                }   
+            }
+
+            // Called periodically when any deck is paused
+            function trackPaused(meta) {
```

Comment:
Is it called periodically while it's paused, or only when the pause happens? The readme says the latter, but this comment says the former